File: src/main/java/com/chattriggers/ctjs/triggers/OnRegularTrigger.java
package com.chattriggers.ctjs.triggers;

public class OnRegularTrigger extends OnTrigger {

    public OnRegularTrigger(Object method, TriggerType triggerType) {
        super(method, triggerType);
    }

    @Override
    public void trigger(Object... args) {
        callMethod(args);
    }
}

File: src/main/kotlin/com/chattriggers/ctjs/triggers/OnRegularTrigger.kt
package com.chattriggers.ctjs.triggers

import com.chattriggers.ctjs.engine.ILoader

class OnRegularTrigger(method: Any, triggerType: TriggerType, loader: ILoader) :
    OnTrigger(method, triggerType, loader) {
    override fun trigger(args: Array<out Any?>) {
        callMethod(args)
    }
}
